geometry_msgs/Point[80] points
